modules/tinymce/src/core/main/ts/keyboard/Nbsps.ts

```diff
@@ -54,6 +55,25 @@ const isAtLineBoundary = (root: SugarElement, pos: CaretPosition) => (
     isBeforeBr(root, pos)
 );
 
+const isCefBlock = (node?: Node) =>
+  Type.isNonNullable(node) && NodeType.isContentEditableFalse(node) && isBlockLike(node);
```

Comment:
I think this affects more than just cef elements I think the space will not be rendered for any block like element. If that is the case then we should make a more general fix and just call isBlockLike without the cef check and rename the function here to `isBlockLikeNode` or something.

Reply:
No, the issue is that the cursor does not go into cef blocks while treewalking, resulting in this issue. There's already things that will do this for other ones, it just didn't do it for cef.